chapter_natural-language-processing/sentiment-analysis-cnn.md

```diff
@@ -322,20 +324,38 @@ def eval_model(features, labels):
     return l_sum / l_n, accuracy.get()[1]
 ```
 
+使用gluon的DataLoader加载数据
+
+```{.python .input  n=100}
+class MyDataset(Dataset):
+    def __init__(self, x, y):
+        super(MyDataset, self).__init__()
+        self.x = x
+        self.y = y
+
+    def __getitem__(self, index):
+        return self.x[index], self.y[index]
+
+    def __len__(self):
+        return len(self.x)
+train_loader = DataLoader(MyDataset(train_features, train_labels), batch_size=batch_size, shuffle=True)
+test_loader = DataLoader(MyDataset(test_features, test_labels), batch_size=batch_size, shuffle=False)
+```
+
 下面开始训练模型。
```

Comment:
Just use gb.train() like in the following example:
http://zh.gluon.ai/chapter_computer-vision/fine-tuning.html

The definitions of gb.train() is available at:
https://github.com/mli/gluon-tutorials-zh/blob/master/gluonbook/utils.py#L371